### ISSUE TITLE
Please rename it to ng-virtual-scroll

### ISSUE THREAD

**Issue by arangates** [rintoj/ngx-virtual-scroller#181]
This is the best Infinite scroll available for angular so far but the name is misleading that it is not the latest one . it is working pretty much awesome in angular 5.1

**Comment by tskweres** [comment]
Does this work in Angular ^5.2.5? All I'm seeing is a blank screen, no viewport items. Can you share a repo or sample that's working with 5+?

**Comment by CoryCharlton** [comment]
@tskweres did you pull the demo code and try that?

**Comment by tskweres** [comment]
Yep it’s working, it was a lazy load / import issue

**Comment by tskweres** [comment]
When I speed scroll I see a lot of white items that take a second or two to populate, other than buffer size is there anything else to combat that?

**Comment by speige** [comment]
@tskweres I don't think the scroll area should have white items for multiple seconds. Can you try the latest version of the npm package & see if it's still happening? If so, could you submit a code sample or a stackblitz?

I've seen this occur when using manual change detection strategy & forgetting to call .refresh() or .detectChanges().

**Comment by exequiel09** [comment]
I was gonna file about the same issue title but mine is of another issue. I'm using this https://github.com/ng-select/ng-select (which uses `[virtualScroll]` as component `@Input`) It causes such conflict between the two when appearing in the same template.

**Comment by speige** [comment]
I agree it should be renamed. I'm waiting on rintoj to give me permissions
over that name in npm.

In the meantime the only workaround I'm aware of is you can create a new
module in your app with a single component called ng-virtual-scroll that
acts as a wrapper around virtual-scroll. Then in your app import that
module & never use the real virtual-scroll, only use the ng-virtual-scroll
wrapper that you made. The wrapper would need a copy/paste of all
input/output property names in the component.ts & the html would just be a
pass-through to virtual-scroll.

Let me know if that works.

**Comment by rintoj** [comment]
Hi @speige, just gave you permission to `ng-virtual-scroll`. The github repo attached to this module is not available now. Please continue to use this repo.

**Comment by speige** [comment]
Thanks. I'll work on this soon.

**Comment by crhistianramirez** [comment]
how about ngx-virtual-scroll? 

That seems to be the trend. x signifying the "version" which can always change. simply ng-virtual-scroll could be confused for angularjs

**Comment by speige** [comment]
@crhistianramirez good point. Will do.

**Comment by speige** [comment]
This has been completed.

ngx-virtual-scroll was already taken, so I chose ngx-virtual-scroller instead

**Comment by alexandrin-rus** [comment]
Thanks for renaming the component. Another issue remains, in the current form it is still conflicting with another popular component https://github.com/ng-select/ng-select that uses the `[virtualScroll]` as an `@Input`. Can the selector also change to `ngx-virtual-scroller , [ngxVirtualScroller]`?

**Comment by speige** [comment]
Oops. You're right, I missed that part. Re-opening issue.

**Comment by speige** [comment]
Ok, this should be fixed now. It's on npm version 1.0.3

I went with virtual-scroller and [VirtualScroller] instead of ngx-virtual-scroller and  [ngxVirtualScroller] for the selectors. I don't think "scroller" will have many conflicts with other libraries & I don't want people to be annoyed by typing the extra ngx everywhere.

**Comment by alexandrin-rus** [comment]
> Ok, this should be fixed now. It's on npm version 1.0.3
> 
> I went with virtual-scroller and [VirtualScroller] instead of ngx-virtual-scroller and [ngxVirtualScroller] for the selectors. I don't think "scroller" will have many conflicts with other libraries & I don't want people to be annoyed by typing the extra ngx everywhere.

That'll work. Thank you!
